index.js

```diff
@@ -31,7 +66,30 @@ function betweenExtremes() {}
  * Example: "A new month"
  * .- / -. . .-- / -- --- -. - ....
  */
-function morseCodeTranslator() {}
+function morseCodeTranslator(message, dictionary) {
+  const words = message.toUpperCase().split(' ');
```

Comment:
Nice - remember this pattern on splitting strings to an array.  We use this a _ton_ in algorithm problems